### PR TITLE
Fix [CON-203] - Color of arrowheads does not match with color of lines

### PR DESCRIPTION
**Pull request by habemuscode**
**Issue**

https://oat-sa.atlassian.net/browse/CON-203

**Related work**

https://github.com/oat-sa/extension-tao-itemqti/issues/1577

**Description**

The color of all arrowheads (including axis arrowheads) changes, when we change the current line or add new. So, it will always inherit the color from the last path created with the arrow-end/ arrow-start attribute set.
 
imagen

The full description of [issue](https://github.com/oat-sa/extension-tao-itemqti)/issues/1577

Also, it looks like the PCI settings are broken on the construct side. When trying to open Lines options an error pops up in the console.

imagen

**AC**

- [x] Fix arrow head color
- [ ] Fix error in console of PCI settings

### CODE REVIEW

**Comment by habemuscode** [comment]
Additional PR for tao -> https://github.com/oat-sa/tao-core/pull/2786

**Review by bziondik — COMMENTED** [review]
I see that in our npm libs we use v2.1.4
https://github.com/oat-sa/tao-item-runner-qti-fe/blob/01c7e2b9b0a6d9ec9921595a2897d7d7b045eca7/package.json#L77
And then use it in different graphic interactions https://github.com/oat-sa/tao-item-runner-qti-fe/blob/01c7e2b9b0a6d9ec9921595a2897d7d7b045eca7/src/qtiCommonRenderer/helpers/Graphic.js#L25
Should we update version also in this libs to have consistency?

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by 404passionFruit — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by habemuscode** [comment]
@bziondik For consistency -> https://github.com/oat-sa/tao-item-runner-qti-fe/pull/144

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: manifest.php [hunk 1]
```diff
@@ -64,7 +64,7 @@
     'label' => 'TAO Base',
     'description' => 'TAO meta-extension',
     'license' => 'GPL-2.0',
-    'version' => '46.14.0',
+    'version' => '46.14.1',
```
Comment (boajer) [inline manifest.php:67]: You've update  from  Raphaël 2.1.2  to 2.2.0, is it more like a bugfix or feature change?

Comment (habemuscode) [reply manifest.php:67]: bugfix

**Comment by habemuscode** [comment]
Updated all npm libraries

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**